### ISSUE TITLE
Docblock and attributes for a class

### ISSUE THREAD

**Issue by iquito** [squizlabs/PHP_CodeSniffer#3286]
Using PSR-12 with `dev-master` (with attribute support), I only encounter one issue in my projects, with a block such as:

```php
namespace project\Controller;

use project\Auth\Authenticate;

/**
 * Shows open tasks for admins
 */
#[Authenticate('admin_logged_in')]
class TodoController extends AbstractController implements MustBeLoggedInInterface
{
}
```

Leads to:

```
 7 | [ERROR | [ ] The file-level docblock must follow the opening PHP
   |        |     tag in the file header
 9 | [ERROR | [x] Header blocks must be separated by a single blank
   |        |     line
```

This should cause no error, as seperating the comment and the attribute with a blank line would be incorrect (as the comment is directly for the class) and it is not a file-level docblock.

This only happens with class docblocks - method docblocks and attributes work fine together.

**Comment by gsherwood** [comment]
Thanks a lot for testing master and reporting this. I've pushed a fix for it.

**Comment by iquito** [comment]
Thanks! Now it works perfectly. Looking forward to 3.6!
